### DISCUSSION TITLE
blog/zh/indie-hacker-tech-stack-2024/

### DISCUSSION THREAD

**Discussion by giscus** [guangzhengli/blog-comments#16] [Announcements]
# blog/zh/indie-hacker-tech-stack-2024/

Ladder@写这篇文章的目的是为了庆祝 X/Twitter 的粉丝突破了 10K 🎉，突然间有了一些分享欲望，在技术栈和云服务厂商的选择如此丰富多样的今天，我们如果需要独立开发的话，应该需要选择什么技术栈呢？

https://guangzhengli.com/blog/zh/indie-hacker-tech-stack-2024/

**Comment by markyun** [comment]
先评再看

**Comment by xingpingcn** [comment]
学到了，感谢分享

**Comment by zzfly256** [comment]
写的不错，思考有深度

**Comment by JasonGaoH** [comment]
点赞

**Comment by Joy0934** [comment]
学到了 写的很好

**Comment by HagerHu** [comment]
👍 整理总结的挺好

**Comment by wp19991** [comment]
"要做独立开发者，独立经营一个产品的话，最应该关注的，应该是尽快交付、尽早收费，不到最后不花钱。"

**Comment by sofakeer** [comment]
感谢分享,很有收获

**Comment by pang-xf** [comment]
写得很棒~

**Comment by brandonxiang** [comment]
数据库用 neon 还是 Supabase

**Comment by tangkunyin** [comment]
写的很不错，有思考有内涵

**Comment by haitaodesign** [comment]
分享的内容很棒，跟我现在搞的事情的看法一致，遇到同行人了

**Comment by countstarss** [comment]
最近一段时间都在youtube上学习Next.js相关的内容，用到的技术栈你上面大部分都提到了，还是得往外看看，不然不知道现在是什么趋势，在学校学习的东西太基础啦

**Comment by SkySung** [comment]
學到很多 尤其是關於Serverless和cloudflare的實際使用心得 謝謝

**Comment by Se1faware** [comment]
很棒的文章！

**Comment by Stargazer1492** [comment]
作者谦虚了，我学到了东西，感谢！

**Comment by tianyw0** [comment]
very nice very helpful

**Reply by tianyw0** [reply]
博主你好，怎么才能获取这篇文章的更新呢 ？

**Comment by Desperado1001** [comment]
学习到了，快速试错，尽早迭代产品，尽早付费。

**Comment by yang-smith** [comment]
给作者点赞。我刚好在考虑独立开发应该如何选择后端，很有价值的文章。个人体验来说，cloudfare worker、vercel的serverless都存在超时问题，10s根本不够用。试着使用fly io，发现还不错，用起来很顺畅。不过从成本和运维来说，可能cloudfare系列是最佳选择。

**Comment by ratup-top** [comment]
很有价值，对于一个全栈开发人员想搭建属于自己的服务很有帮助

**Comment by ChrisPei** [comment]
写的不错，支持

**Comment by AltarIbnL** [comment]
作者写的很好，希望以后能有更多更好的分享

**Comment by houweinuo** [comment]
提个小建议，站内的所有的链接跳转 可以设置成新开页面 这样交互会好很多

**Comment by jacky-xbb** [comment]
做独立开发快一年了，我现在用的也是 next.js + react，因为我自己有 rails 基础，想尝试用 rails + hotwire + rails 开发 AI 相关的 SAAS，博主有啥建议么？

**Reply by guangzhengli** [reply]
如果会 nextjs 的话，建议就 vercel ai sdk 进行二次开发，有真正用户了再用 ruby 写点后端也行，毕竟 ruby 暂时还没有好的AI 框架可以二次开发呀

**Comment by shuaipotian-joke** [comment]
笔者的分享很用心也很有深度,感觉这就是我一直在寻找的路和确定的方向,确定好起点就该出发,后续再来回顾

**Comment by JeffAI-dev** [comment]
博主和各位观众姥爷,
我们也可以合作,一起接单,
我是一名AI开发者, 在线接单. 
微信: bitcraftsmancj

**Comment by wangdabaoqq** [comment]
你超棒的！

**Comment by xyang0917** [comment]
多谢分享

**Comment by WWWeedss** [comment]
很有收获～

虽然在这里问感觉不太好，但是还是很好奇，各位独立开发者的 Idea 都是从哪里冒出来哒？自从我上班且养成极其规律的生活作息之后，感觉自己感知问题与需求的能力都消失殆尽了。 T.T

**Reply by pang-xf** [reply]
大部分来自于自身的需求 做自己产品的活跃用户

**Comment by highestpeak** [comment]
感谢分享，条理清晰，逻辑清楚，给了很多 insight

**Reply by highestpeak** [reply]
另外我有个困惑：到底多快的交付才算“快”？还有一个问题是，我怎么判断产品发展到某个阶段，比如 xxx，就应该交付一个版本了？这主要是靠经验判断吗？其实我自己在这方面还有点“完美主义”，虽然也在努力克服，但总觉得很难把握。博主有没有这方面的经验可以分享一下呢？

**Comment by hellozhangran** [comment]
学到了，收获很大，感谢分享🙏

**Comment by OptimisticQuan** [comment]
写到最后，笔者忽然认为自己并没有什么实际的成功经验
哈哈哈

总结的特别好，感谢分享

**Comment by lizhijun** [comment]
感谢整理分享，非常实用
